### DISCUSSION TITLE
Any way to import .css file from old versions to v3.3

### DISCUSSION THREAD

**Discussion by ekrocks** [bullhorn/career-portal#421] [Q&A] [answered]
I have been trying to upgrade the current OSCP to the newest version available (version 3.3) but having some problems regarding it. Currently the appearance of the website is stylized to our company's visuals, and I would like to know if there are any way for me to basically import the appearance instead of coding everything again.

The current version of the OSCP is quite old (hasn't been updated in years with the last modified date being September of 2016) and I noticed that there are a lot of differences on how .css file is created. The server currently holds 2 .css files that have very odd names like (app-14de02fad9.css and vendor-f398fee98b.css) and how the files are structured seems very different than the 3.3 version. 

The current OSCP looks like this: 
![image](https://user-images.githubusercontent.com/10538258/133996773-c0ed78f3-7e1a-4402-8558-868d9af5a781.png)
![image](https://user-images.githubusercontent.com/10538258/133996838-41340aa9-3d6e-4996-9e54-d0230f209384.png)
 
and I have tried moving the current .css files to custom.css on version 3.3 but it seems to not do much other than break bunch of icons like such: 
![image](https://user-images.githubusercontent.com/10538258/133997447-903ff9de-bbcf-40f2-bb07-10f8399bfc25.png)
![image](https://user-images.githubusercontent.com/10538258/133997527-543ec4d2-8a50-46e7-a97f-1d25142812af.png)
![image](https://user-images.githubusercontent.com/10538258/133997624-29bb9491-1414-4344-9bf0-fe95cf96f6f5.png)

**Comment by charlesabarnes** [comment, marked as answer]
Hello, This newer versions (v3+) of the career portal was built on Bullhorn's [novo-elements](https://github.com/bullhorn/novo-elements).   There is not a 1:1 conversion for styling to make it look identical to previous versions of the career portal as there is some differing functionality and layout.  The customizations probably would have to be specific to this version of the career portal.   

Instead of working from the released version of the career portal, you can perhaps have some ease in customization by cloning the career portal, overriding some of the scss variables, and customizing it that way.

**Reply by ekrocks** [reply]
Ok got it. Thank you
